### DISCUSSION TITLE
10.8.12

### DISCUSSION THREAD

**Discussion by joshuaboniface** [jellyfin/jellyfin#10535] [Announcements]
# Jellyfin 10.8.12

Stable hotfix release for 10.8.z release branch.

## New Features and Major Improvements

N/A

## Release Notes

N/A

## Changelog

GitHub Project: https://github.com/orgs/jellyfin/projects/29

### [jellyfin](https://github.com/jellyfin/jellyfin) [5]

Note: Dependabot & Renovate automatic PRs are excluded from this list.

 * #10528 [@nyanmisaka] Fix mismatched intel VAAPI UMD/KMD - Backport #10451
 * ~~#10514 [@Bond-009] Update SkiaSharp (v2.88.6) and BlurHashSharp (v1.3.1)~~ Reverted due to SEGV on access
 * #10487 [@thornbill] Fix scaleFactor limit - Backport #9738
 * #10454 [@Shadowghost]  Add MALLOC_TRIM_THRESHOLD_ to default ENV
 * #10406 [@thornbill] Fix playlist listings

### [jellyfin-web](https://github.com/jellyfin/jellyfin-web) [6]

Note: Dependabot & Renovate automatic PRs are excluded from this list.

 * jellyfin/jellyfin-web#4912 [@dmitrylyzo] Fix imports
 * jellyfin/jellyfin-web#4893 [@thornbill] Change Subtitle Sync slider to go from -300 to 300 - Backport #4754
 * jellyfin/jellyfin-web#4892 [@thornbill] Fix language/subtitle switcher when using gamepad - Backport #4860
 * jellyfin/jellyfin-web#4837 [@dmitrylyzo] Fix playing empty item set
 * jellyfin/jellyfin-web#4813 [@dmitrylyzo] Handle pressing Enter to finish keyboard dragging of slider (Backport #4805)
 * jellyfin/jellyfin-web#4797 [@Mauroq] Remove H264 10bit support on Samsung TV (Tizen)

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin/releases/tag/v10.8.12'>10.8.12</a>.</em>

**Comment by xheavenx92** [comment]
Hello there, it is a way to implement a transcode to UTF-8 for subs into the app ?

**Comment by notjames** [comment]
Hi! I'm trying to figure out what the correct directory structure is for TV Shows. Please see: https://forum.jellyfin.org/t-tv-shows-directory-structure?pid=10057#pid10057

This is really driving me crazy and seriously the way TV shows are indexed seems buggy to me. I just found https://jellyfin.org/docs/general/server/media/shows/ and I'll give that a go but I would VERY much like to be able to index it similar to how my movies are indexed IE with the TV rating and year of the show as its own directory.

**Reply by Shadowghost** [reply]
The scanner is limited to the directory structure(s) described in the docs. Might change in the future but there is nothing specific planned at the moment.
